src/Compilers/Core/MSBuildTaskTests/IntegrationTests.cs

```diff
@@ -26,7 +26,10 @@ public class IntegrationTests : TestBase
         static IntegrationTests()
         {
             s_msbuildDirectory = TestHelpers.GetMSBuildDirectory();
-            s_msbuildExecutable = Path.Combine(s_msbuildDirectory, "MSBuild.exe");
+            if (s_msbuildExecutable != null)
+            {
+                s_msbuildExecutable = Path.Combine(s_msbuildDirectory, "MSBuild.exe");
+            }
```

Comment:
So if `s_msbuildExecutable` is `null` we leave it alone, but if it's not `null` we explicitly overwrite it? Why?